packages/web-config-server/src/apiV1/dataBuilders/generic/table/tableOfDataValues/getValuesByCell.js

```diff
@@ -68,7 +68,12 @@ export const getPercentageCountOfValuesByCell = (cells, results) => {
 export const getCalculatedValuesByCell = (cells, results) => {
   const calculatedValuesByCell = {};
   cells.forEach(cell => {
-    calculatedValuesByCell[cell.key] = calculateArithmeticOperationForAnalytics(results, cell);
+    if (typeof cell === 'string') {
```

Comment:
I use `typof cell === 'string'` a few times throughout this PR. I find it easier to read than `!cell.someProperty`, which is used some other places in the code. Do you agree or is there a reason that `typeof` is avoided?